src/components/meetups/Meetupitem.js

```diff
@@ -0,0 +1,27 @@
+import Card from '../ui/Card';
+import classes from './MeetupItem.module.css';
+
+function MeetupItem(props){
+    return (
+    <div className={classes.display}>
+    <li className={classes.item}>
+        <Card>
+        <div className={classes.image}>
```

Comment:
children of a component should be indented, please fix it to be more like:

```jsx
<Card>
    <div></div>
</Card>
```

not 

```jsx
<Card>
<div></div>
</Card>
```